lib/screens/home_screen.dart

```diff
@@ -14,35 +14,56 @@ class _HomeScreenState extends State<HomeScreen> {
   Widget build(BuildContext context) {
     return Scaffold(
       appBar: AppBar(
-        title: const Text("Easy Scan"),
+        backgroundColor: primaryColor,
+        title: const Text(
+          "Easy Scan",
+          style: TextStyle(
+              fontSize: 24, fontWeight: FontWeight.bold, color: Colors.white),
+        ),
         centerTitle: true,
       ),
-      body: Column(
-        crossAxisAlignment: CrossAxisAlignment.stretch,
-        children: [
-          HomeCard(
-            onTap: () => Navigator.push(
-                context, MaterialPageRoute(builder: (_) => ScanAndConvert())),
-            iconData: Icons.image_search,
-            color: Colors.red,
-            text: 'Scan and convert',
-          ),
-          HomeCard(
-            onTap: () => Navigator.push(
-                context, MaterialPageRoute(builder: (_) => ImageToPdf())),
-            iconData: Icons.picture_as_pdf_outlined,
-            color: Colors.green,
-            text: 'Images to Pdf',
-          ),
-          HomeCard(
-            onTap: () => Navigator.push(
-                context, MaterialPageRoute(builder: (_) => SavedPdfScreen())),
-            iconData: Icons.save_rounded,
-            color: Colors.amber,
-            text: 'History',
-          ),
-        ],
-      ),
+      body: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
+        SizedBox(
+          height: MediaQuery.of(context).size.longestSide * 0.2,
+        ),
+        HomeCard(
+          onTap: () => Navigator.push(
+              context, MaterialPageRoute(builder: (_) => ScanAndConvert())),
+          iconData: Icons.image_search,
+          text: 'Scan and Convert',
+        ),
+        HomeCard(
+          onTap: () => Navigator.push(
+              context, MaterialPageRoute(builder: (_) => ImageToPdf())),
+          iconData: Icons.picture_as_pdf_outlined,
+          text: 'Pick and Export',
+        ),
+        // This widget is needed to be implemented for recent projects
```

Comment:
Adapt with the latest code i.e history is missing here

Reply:
please elaborate.

Reply:
@nayakastha compare with older version [here](https://github.com/askbuddie/easy-scan/pull/30/files)